model/ftn/w3fld1md.ftn

```diff
@@ -120,6 +123,7 @@
 !       RIB     REAL   I   Bulk Richardson in lower atmosphere
 !                          (for determining stability in ABL to get
 !                          10 m neutral wind)
+!       DAIR    REAL   I   Air density
```

Comment:
For both w3fld1md and w3fld2md you have added DAIR but you aren't actually using it, so I would not update this and leave it as is until you are actually using DAIR in the calculations.

Reply:
In both routines there are operations using DAIR, for example:

https://github.com/ukmo-waves/WW3/blob/a157dc91c362634552251dbd62aab8d5cf9f6dfb/model/ftn/w3fld1md.ftn#L380-L382

The changes I made aim to using DAIR as obtained from a routine input parameter, rather than using the constant value of DAIR stored in the CONSTANTS module.

Reply:
I see this now.  I was just looking at the changed code, not the whole code.  Thanks for pointing this out!